### PR TITLE
Add comments to script

### PR DESCRIPTION
**Pull request by garibaldiviolin**
# Reason
Current scripts doesn't have any comment.

# What was done
Some comments were added to the script.

### CODE REVIEW

File: aiohttp_test.py [hunk 1]
```diff
@@ -29,6 +30,7 @@ def print_responses(result):
     print(result)
 
 
+# Main application
```
Comment (testeapi2) [inline aiohttp_test.py:33]: This comment is very short.

File: aiohttp_test.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# aiohttp_test.py
```
Comment (testeapi2) [inline aiohttp_test.py:1]: This one is OK.

**Review by testeapi2 — COMMENTED** [review]
Review done.

File: aiohttp_test.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 from aiohttp import ClientSession
```
Comment (testeapi2) [inline aiohttp_test.py:2]: Look at this.

Comment (garibaldiviolin) [reply aiohttp_test.py:2]: I think it's ok:
`import this`

Comment (garibaldiviolin) [reply aiohttp_test.py:2]: > Look at this.

Test

**Pull request merged**